.changeset/polite-ligers-hang.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/plugin-techdocs': patch
+'@backstage/plugin-techdocs-addons-test-utils': patch
```

Comment:
That's not really the change to this package though? 😅 Probably best to split this into two changesets